src/components/Buttons/ButtonForm/ButtonForm.tsx

```diff
@@ -0,0 +1,10 @@
+import { ButtonProps } from '../../../types/types';
+import './ButtonForm.scss';
+
+const ButtonForm = ({ text }: ButtonProps) => (
```

Comment:
Logically, it should be named SubmitButton of FormButton, but not ButtonForm because it means you built form for buttons